client/my-sites/jetpack-search/upsell.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import FormattedHeader from 'calypso/components/formatted-header';
+import PromoCard from 'calypso/components/promo-section/promo-card';
+import PromoCardCTA from 'calypso/components/promo-section/promo-card/cta';
+import useTrackCallback from 'calypso/lib/jetpack/use-track-callback';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import WhatIsJetpack from 'calypso/components/jetpack/what-is-jetpack';
+
+/**
+ * Asset dependencies
+ */
+import JetpackSearchSVG from 'calypso/assets/images/illustrations/jetpack-search.svg';
+
+export default function JetpackSearchUpsell(): ReactElement {
+	const onUpgradeClick = useTrackCallback( undefined, 'calypso_jetpack_search_upsell' );
+	const siteSlug = useSelector( getSelectedSiteSlug );
+
+	return (
+		<Main className="jetpack-search">
+			<DocumentHead title="Jetpack Search" />
+			<SidebarNavigation />
+			<PageViewTracker path="/jetpack-search/:site" title="Jetpack Search" />
+
+			<FormattedHeader
+				headerText={ translate( 'Jetpack Search' ) }
+				id="jetpack-search-header"
+				align="left"
+				brandFont
+			/>
+
+			<PromoCard
+				title={ translate( 'Finely-tuned search for your site.' ) }
+				image={ { path: JetpackSearchSVG } }
+				isPrimary
+			>
+				<p>
+					{ translate(
+						'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
+					) }
+				</p>
+
+				<PromoCardCTA
+					cta={ {
+						text: translate( 'Upgrade to Jetpack Search' ),
+						action: {
+							url:
+								'https://jetpack.com/upgrade/search/?utm_campaign=my-sites-jetpack-search&utm_source=calypso&site=' +
```

Comment:
I've linked to the Upgrade page rather than directly to the checkout because it gives visitors the opportunity to learn more about the product before buying.

Reply:
Sounds good to me.